docs-src/silent-refresh.md

```diff
@@ -1,6 +1,6 @@
 ## Refreshing when using Implicit Flow (Implicit Flow and Code Flow)
 
-**Notes for Code Flow**: You can also use this strategy for refreshing tokens when using code flow. However, please note, the strategy described within [Token Refresh](./token-refresh.md) is far easier in this case.
+**Notes for Code Flow**: You can also use this strategy for refreshing tokens when using code flow. However, please note, the strategy described within [Token Refresh](./token-refresh.html) is far easier in this case.
```

Comment:
not sure about changing the extension here, it should be automatically transformed by the MD to HTML parser

Reply:
This is also what I thought initially. Thought that it is enough to include it in `summary.json` so that the page is generated and then when generating the html files, the .md references will be replaced by the respective .html-pages. 

However this didn't work out, so I had a look at other places where pages are referenced. For instance in `docs-src/popup.md` you will find this line: `- Create and configure a ``silent-refresh.html`` as described [here](./silent-refresh.**html**) *.`

There are also other places where it's handled like this (e.g. in `docs-src/session-checks.md`).

I guess the parser is not able to replace it (will read about it